File: modul-02/bahasa c/PRAK205-2310817310003-Mfaridsetiawan.c
#include <stdio.h>
#include <math.h>
int main(){
    int a, b;
scanf("%d", &a);
scanf("%d", &b);

int alas = sqrt(pow(b,2)-(pow(a,2)));
int tinggi = a;
int keliling = a + b + alas;
int luas = 0.5 * alas * tinggi;

printf("\n");
printf("Alas= %d cm \n", alas);
printf("tinggi= %d cm \n", tinggi);
printf("keliling= %d cm \n", keliling);
printf("Luas= %d cm^2 \n", luas);

return 0;
}

File: modul-02/python/PRAK205-2310817310003-Mfaridsetiawan.py
import math

a = input().split()

if len(a) == 1:
     a = int(a[0])
     b = int(input())
elif len(a) == 2:
     a, b = map(int,a)

alas = int(math.sqrt(b**2 - a**2))
tinggi = a
keliling = a + b + alas
luas = 0.5 * alas * tinggi

print(f"Alas = {alas} cm")
print(f"Tinggi = {tinggi} cm")
print(f"Keliling = {keliling} cm")
print(f"Luas = {luas:.0f} cm^2")